packages/suite/src/views/wallet/receive/components/ConnectDevicePromo.tsx

```diff
@@ -33,6 +33,13 @@ const ConnectDevicePromo = ({ title, description }: ConnectDevicePromoProps) =>
     );
 };
 
+export const ConnectDeviceGenericPromo = () => (
+    <ConnectDevicePromo
+        title={<Translation id="TR_RECONNECT_HEADER" />}
+        description={<Translation id="TR_CONNECT_DEVICE_SEND_PROMO_TITLE" />}
```

Comment:
I would love different text here. Something like `To perform trade, connect your Trezor` but validate it with Ben or Chris